sources/web/datalab/polymer/components/datalab-editor/datalab-editor.html

```diff
@@ -47,7 +44,7 @@
       }
       #editorToolbar {
         display: flex;
-        flex: 0 0 var(--toolbar-height);
+        height: calc(var(--toolbar-height) - 1px); /* for border-bottom */
```

Comment:
Is it worth using a variable for the border thickness?

Reply:
I don't think these two occurrences warrant a variable, so I'll merge this as is, and we can revisit if you think it's much cleaner.